cumulus/polkadot-parachain/src/chain_spec/glutton.rs

```diff
@@ -28,26 +27,23 @@ pub fn glutton_westend_config(
 	let mut properties = sc_chain_spec::Properties::new();
 	properties.insert("ss58Format".into(), 42.into());
 
-	let authorities = match chain_type {
-		ChainType::Development => vec![Sr25519Keyring::Alice.public().into()],
-		// Even the westend live setup does currently use Alice & Bob.
-		_ => vec![Sr25519Keyring::Alice.public().into(), Sr25519Keyring::Bob.public().into()],
-	};
-
 	GenericChainSpec::builder(
 		glutton_westend_runtime::WASM_BINARY.expect("WASM binary was not built, please build it!"),
 		Extensions { relay_chain: relay_chain.into(), para_id: para_id.into() },
 	)
 	.with_name(&chain_type_name(para_id, &chain_type))
 	.with_id(&chain_id(para_id, &chain_type))
-	.with_chain_type(chain_type)
-	.with_genesis_config_patch(
-		glutton_westend_runtime::genesis_config_presets::glutton_westend_genesis(
-			authorities,
-			Some(Sr25519Keyring::Alice.to_account_id()),
-			para_id,
-		),
-	)
+	.with_chain_type(chain_type.clone())
+	.with_genesis_config_preset_name(match chain_type {
+		ChainType::Development => sp_genesis_builder::DEV_RUNTIME_PRESET,
+		ChainType::Local => sp_genesis_builder::LOCAL_TESTNET_RUNTIME_PRESET,
+		_ => panic!("chain_type: {chain_type:?} not supported here!"),
```

Comment:
```suggestion
		_ => sp_genesis_builder::LOCAL_TESTNET_RUNTIME_PRESET,
```

Was before the same.

Reply:
Before what? The last change from my PR https://github.com/paritytech/polkadot-sdk/pull/7481/files#diff-2f041ef228e62fc556043213a729de8728b114e688112e710ffebce46bbbd25a, has merged the different functions into one, and we supply the chain-type to distinguish the preset now.